GrassChoppers/Modules/Home/HomeLogicController.swift

```diff
@@ -1,11 +1,12 @@
 protocol HomeLogicControlling: class {
     var delegate: HomeLogicControllingDelegate? { get set }
     
+    func viewDidLoad()
     func buttonPressed()
 }
 
 protocol HomeLogicControllingDelegate: class {
-    
+    func update(name: String)
```

Comment:
Here we are defining what it means to be a delegate of the `HomeLogicController`. Same concept as with the [data manager](https://github.com/jwitcig/GrassChoppers/pull/2/files#r463916256). We're just saying that, to be a delegate for the logic controller (which in practice will be the ViewController), you need to have this function.